http_output.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import( "fmt"
+	"bytes"
+	"log"
+	"net/http"
+	"time"
+	"os"
+	"os/signal"
+	"syscall"
+	"crypto/tls"
+	"strings"
+)
+
+type HttpFormatter interface {
+	Format( string ) string
+}
+
+/* This is an implementation of the HttpFormatter interface which is specific to the SensorAlertsApi */ 
+type SensorAlertsFormatter struct {
+}
+
+/* this function takes the json string specified by the event parameter and transforms it into a format
+   which the SensorAlertsApi can consume */
+func ( this *SensorAlertsFormatter ) Format( event string ) string {
+	return `{ "service" : "carbonblack", "alerts" : [ ` + event + ` ] }`
```

Comment:
`text/template` is the name of the golang stdlib package that provides templating: see https://golang.org/pkg/text/template/